styles.css

```diff
@@ -0,0 +1,322 @@
+}
+.choose h1 {
+font-family: "urbanist";
+font-weight: 500;
+font-size: 18px;
+line-height: 32px;
+text-align: center;
+color:#737373;
+}
+.listcard {
+display: flex;
+justify-content: center;
+gap: 40px;
+width:  100%;
+height: 400px;
+padding: 50px;
+} 
+.card {
+ border: solid #fdfafa;
+background-color: #fff;
+}
+.card-1 {
+background-color: #25A59E;
+border: solid #25A59E;
+}
+.listtop {
+width: 100%;
+height:  4px;
+top: 0;
+left: 0;
+background-color: #25A59E;
+}
+.title {
+font-family: "Gilda display";
+font-weight: 400;
+font-size: 30px;
+line-height: 62px;
+padding-left: 30px;
+}
+.title-1 {
+font-family: "Gilda display";
+font-weight: 400;
+font-size: 30px;
+line-height: 62px;
+color: #fff;
+padding-left: 30px;
+}
+.description {
+font-family: "urbanist";
+font-weight: 400;
+font-size: 12px;
+line-height: 32px;
+color: #737373;
+padding-left: 30px;
+}
+.description-1 {
+font-family: "urbanist";
+font-weight: 400;
+font-size: 12px;
+line-height: 32px;
+color: #fff;
+padding-left: 30px;
+}
+.action {
+display: flex;
+gap: 7px;
+margin-left: 230px;
+}
+.text {
+font-family:"Gilda display";
+font-weight: 400;
+font-size: 13px;
+line-height: 23.54px;
+}
+.action-1 {
+display: flex;
+gap: 5px;
+margin-left: 230px;
+}
+.text-1 {
+font-family:"Gilda display";
+font-weight: 400;
+font-size: 13px;
+line-height: 23.54px;
+color: #fff;
+}
+.span-1 {
+color:#25A59E;
+}
+.journey {
+font-family: "Gilda display";
+font-weight: 400;
+font-size: 40px;
+line-height: 50px;
+color: #262626;  
+transform: translateY(50%);  }
+.thousands {
+font-family: urbanist;
+font-weight: 400;
+font-size: 12px;
+margin-right: 50px;
+line-height: 20px;
+margin-top: 30px;
+}
+.span-2 {
+color:#25A59E;
+} 
+.couples {
+transform: translateY(-100%);
+margin-left: 100px;
+}
+.arrow {
+color: #25A59E;
+transform: translateX(35%)
+}
+.img-group {
+margin-left: 400px;
+margin-top: -250px
+}
+.trip  {
+font-family: "Gilda display";
+font-weight: 400;
+font-size: 3.5vw;
+line-height: 64px;
+height: 168px;
+text-align: center;
+color: #262626;
+margin-left: 50px;
+}
+.video {
+width: 100%;
+}
+.header-container {
+font-family: "Gilda display";
+font-weight: 400;
+font-size: 30px;
+display: flex;
+gap: 500px;
+}
+.click {
+transform: translateX(60%);
+margin-left: 5px;
+}
+.move {
+font-family: "Gilda Display";
+font-size: 28px;
+font-weight: 400;
+line-height: 100px;
+letter-spacing: 0.1em;
+text-align: left;
+color: #25A59E;
+}
+.agent {
+font-family: "Urbanist";
+font-size: 18px;
+font-weight: 500;
+line-height: 32px;
+text-align: left;
+margin-bottom: 50px;
+color: #595959;
+}
+.social-link {
+display: flex;
+gap: 10px;
+}
+.bird {
+margin-left: -150px;
+}
+.youtube {
+margin-left: -70px;
+}
+.notification {
+margin-top: 30px;
+}
+.header-links h3 {
+display: inline;
+margin: 0 10px;
+font-family: "Urbanist";
+font-size: 18px;
+font-weight: 600;
+color: #595959;
+}
+.header-links {
+margin-left: 600px;
+}
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+      
```

Comment:
Hi @musyelevateher ,

Your project is complete! There is nothing else to say other than... it's time to merge it!!!
Congratulations! 🎉

<div style="text-align:center;">
    <img src="https://media1.giphy.com/media/gLu90OMjz4j3hAkJk2/giphy.gif?cid=47028fa8sx12d824xrvy8zo8bq3w00f23hoj11gqqzus7q0f&ep=v1_gifs&rid=giphy.gif&ct=g" alt="Remote Collaboration" style="width:100%;">
</div>

### Highlights
- ✅ **Professional Readme**
- ✅ **Good use of Gitflow**
- ✅ **UI looks like the Figma design**
- ✅ **Good use of HTML semantics**
- ✅ **Professional commit messages**
- ✅ **Professional PR title**
- ✅ **Professional PR description**

### Optional Suggestions
Every comment with the `[OPTIONAL]` prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding! 👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.  
Please, remember to tag me in your question so I can receive the notification._